aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.kt

```diff
@@ -82,7 +82,7 @@ class AWSCognitoAuthPlugin : AuthPlugin<AWSCognitoAuthServiceBehavior>() {
                 configuration,
                 AWSCognitoAuthServiceBehavior.fromConfiguration(configuration),
                 credentialStoreClient,
-                configuration.userPool?.let { UserContextDataProvider(context, it) },
+                configuration.userPool?.let { UserContextDataProvider(context, it.poolId!!, it.appClient!!) },
```

Comment:
Any way to safe-unwrap here?

Reply:
I have a follow up PR for refactoring all the `!!` and `?: ""`.